meshroom/ui/qml/main.qml

```diff
@@ -838,11 +838,17 @@ ApplicationWindow {
                 reconstruction: _reconstruction
                 readOnly: _reconstruction.computing
 
-                function viewAttribute(attribute, mouse) {
-                    let viewable = false;
-                    viewable = workspaceView.viewIn2D(attribute);
-                    viewable |= workspaceView.viewIn3D(attribute, mouse);
-                    return viewable;
+                function viewNode(node, mouse) {
+                    // 2D viewer
+                    viewer2D.displayedNode = node;
+
+                    // 3D viewer
+                    for(var i=0; i < node.attributes.count; ++i)
```

Comment:
The spaces here for your "for"/"if" (and for var i = 0) are not coherent with what you used everywhere else (but that's nitpicking).